metrics-proxy/src/main/java/ai/vespa/metricsproxy/http/application/ApplicationMetricsHandler.java

```diff
@@ -64,4 +79,16 @@ private JsonResponse applicationMetricsResponse(String requestedConsumer) {
         }
     }
 
+    private TextResponse applicationPrometheusResponse(String requestedConsumer) {
+        ConsumerId consumer = getConsumerOrDefault(requestedConsumer, metricsConsumers);
+        var metricsByNode = metricsRetriever.getMetrics(consumer);
+
+
+        return new TextResponse(200, PrometheusUtil.toPrometheusModel(toGenericApplicationModel(metricsByNode).nodes.stream()
```

Comment:
You can add a static import of the `toPrometheusModel`, as the class name does not add any extra value, just extra characters.